### PR TITLE
Completion of basic journey

### PR DESCRIPTION
**Pull request by garethgooch**
basic nav, basic CYA, removed packages and docs from CYA as not required at this stage

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -776,6 +776,7 @@ consigneeForAllItems.error.required = Select yes if you are the consignee for al
 traderDetailsConsigneeAddress.title = What is the consignee’s address for item {0}
 traderDetailsConsigneeAddress.heading = What is {0}’s address?
 traderDetailsConsigneeAddress.checkYourAnswersLabel = {0}'s address
+traderDetailsConsigneeAddress.checkYourAnswersLabel.fallback = Consignee
```
Comment (chris-ianson) [inline conf/messages.en:779]: What's fallback for?

Comment (garethgooch) [reply conf/messages.en:779]: on the CYA if the user is able to get to it without selecting a consignor/nee name it will pull this back in the getOrElse

**Review by chris-ianson — APPROVED** [review]
Think addItemsSameConsignorForAllItems and Consignee row are used add items CYA

I'm happy with the comments and TODOs, we know this is going to expand as more sections become available.

Do the journey tests need updating before this goes in?

**Comment by garethgooch** [comment]
Journey tests pass as they are not reliant on CYA yet

**Pull request merged**